sql-cloudant/src/main/scala/org/apache/bahir/cloudant/common/ChangesRow.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2017 IBM Cloudant. All rights reserved.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+package org.apache.bahir.cloudant.common;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+
+import java.util.List;
+
+/**
+ * Class representing a single row in a changes feed. Structure:
+ *
+ * {
+ *   last_seq": 5
+ *   "results": [
+ *     ---*** This next items is the ChangesRow ***---
+ *     {
+ *       "changes": [ {"rev": "2-eec205a9d413992850a6e32678485900"}, ... ],
+ *       "deleted": true,
+ *       "id": "deleted",
+ *       "seq": 5,
+ *       "doc": ... structure ...
+ *     }
+ *   ]
+ * }
+ */
+public class ChangesRow {
```

Comment:
@emlaver -- Java sources should reside under `src/main/java` not `src/main/scala` unless you can convert this code to Scala (preferably)

Reply:
Fixed in 5da88e8.